odinbot/agent.py

```diff
@@ -182,33 +182,17 @@ async def _create_agent(self) -> AnyAgent:
             channel_id=self.channel_id
         )
         
+        MCP_DISCORD_URL = os.environ.get('MCP_DISCORD', 'http://localhost:8080/mcp')
+        logger.info(f"{MCP_DISCORD_URL=}")
         return await AnyAgent.create_async(
             "openai",
             AgentConfig(
                 model_id="o3",
                 instructions=instructions,
                 tools=[
-                    MCPStdio(
-                        command="docker",
-                        args=[
-                            "run",
-                            "-i",
-                            "--rm",
-                            "-e",
-                            "DISCORD_TOKEN",
-                            "mcp/mcp-discord",
-                        ],
-                        env={
-                            "DISCORD_TOKEN": os.getenv("DISCORD_TOKEN"),
-                        },
-                        tools=[
-                            "test",
-                            "discord_read_messages",
-                            "discord_login",
-                            "discord_send",
-                        ],
-                        client_session_timeout_seconds=60.0,  # Increased timeout to 60 seconds
-                    ),
+                    MCPSse(
```

Comment:
TODO before merging (and why this is a draft):
```suggestion
                    MCPStreamableHttp(
```

Pending [this nice update](https://github.com/mozilla-ai/any-agent/issues/488).

Reply:
Aaand there we are. Any Agent supports streamable http now, so we can do this.